test/TestProjects/CadlPetStore/Generated/ToyListResults.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using Azure.Core;
+
+namespace CadlPetStore
+{
+    public partial class ToyListResults
+    {
+        public object Items { get; }
```

Comment:
Should this be a `IReadOnlyList<T>`?

Reply:
Is this related to https://github.com/Azure/autorest.csharp/issues/2471?  @archerzz 
Seems this part is not implemented yet.

Reply:
> Is this related to #2471? @archerzz Seems this part is not implemented yet.

I think so.